pkg/activator/handler/handler_test.go

```diff
@@ -781,6 +805,15 @@ func serviceLister(svcs ...*corev1.Service) corev1listers.ServiceLister {
 	return services.Lister()
 }
 
+func setupConfigStore(t *testing.T) (*activatorconfig.Store, *corev1.ConfigMap) {
+	tracerUpdater := configmap.TypeFilter(&tracingconfig.Config{})(func(name string, value interface{}) {
```

Comment:
Why do we create this updater? I don't think it's needed to create the store as the argument is variadic

Reply:
That should be correct -- thanks! still getting used to golang :)